app/views/users/edit.html.erb

```diff
@@ -22,18 +22,16 @@
 <header class="crayons-layout crayons-layout--limited-l crayons-layout--1-col p-4">
   <% if @tab == "organization" && @organizations.present? %>
     <h1 class="crayons-title">Settings for
-      <span class="organization-dropdown">
-        <select id="organization-dropdown-select" onchange="window.location.href=this.value">
-          <% options = @organizations.pluck(:slug, :id).each do |arr| %>
-            <% arr[0] = "@#{arr[0]}" %>
-            <% arr[1] = "/settings/organization/#{arr[1]}" %>
-          <% end.to_h.merge("New Organization" => "/settings/organization/new") %>
-          <%= options_for_select(options, "/settings/organization/#{params[:org_id]}") %>
-          <%# For example: %>
-          <%# <option value="/settings/organization/1>@devteam</option> %>
-          <%# <option value="/settings/organization/new" selected>New Organization</option> %>
-        </select>
-      </span>
+      <select class="crayons-select lh-tight w-25 align-text-top" onchange="window.location.href=this.value">
```

Comment:
```suggestion
      <select class="crayons-select lh-tight w-25 align-text-top w-max" onchange="window.location.href=this.value">
```

Adding `w-max` to the `select` element will show the entire string, especially on a smaller screen.

### Before (without `w-max`)
![Screen Shot 2020-11-03 at 1 42 02 PM](https://user-images.githubusercontent.com/816402/98043745-97b17e80-1dda-11eb-90b7-88ce895d1a93.png)

### After (with `w-max`)
![Screen Shot 2020-11-03 at 1 41 50 PM](https://user-images.githubusercontent.com/816402/98043738-95e7bb00-1dda-11eb-9318-19c5510a4626.png)